ChangeLog

```diff
@@ -26,6 +26,10 @@ Release date: TBA
 
   Closes #104
 
+* Module (``%``) operations on strings are now correctly inferred.
```

Comment:
I think we should call it `%-formatting` to avoid sounding like the mathematical modulo operator.